health/health.d/geth.conf

```diff
@@ -0,0 +1,25 @@
+ template: chainhead_diff_between_header_block
+       on: geth.chainhead
+    class: Workload
+     type: ethereum_node
+component: Geth
+    every: 5s
```

Comment:
```suggestion
    every: 10s
```
Usually, we use 10s as the default minimum.

<details>
  <summary>stock alarms every [click!]</summary>
  
```cmd
[ilyam@pc health.d]$ grep -E -o -h  "every.*" -R * | sort | uniq -c | sort -n -r
    173 every: 10s
     79 every: 1m
     12 every: 30s
     10 every) : (60 * $update_every))
     10 every) : ( 5 * $update_every))
      5 every: 60s
      3 every: 5s
      3 every: 5m
      3 every: 20s
      3 every 1 minute
      2 every: 60
      1 every 10 seconds
```

</details>